src/gui/properties/trackerlist.cpp

```diff
@@ -524,13 +556,65 @@ void TrackerList::showTrackerListMenu(QPoint) {
   }
 }
 
-void TrackerList::loadSettings() {
-  if (!header()->restoreState(Preferences::instance()->getPropTrackerListState())) {
-    setColumnWidth(0, 30);
-    setColumnWidth(1, 300);
-  }
+void TrackerList::loadSettings()
+{
+    header()->restoreState(Preferences::instance()->getPropTrackerListState());
+}
+
+void TrackerList::saveSettings() const
+{
+    Preferences::instance()->setPropTrackerListState(header()->saveState());
+}
+
+QStringList TrackerList::headerLabels()
+{
+    QStringList header {
```

Comment:
@Chocobo1, then we can't use tr() (static QObject::tr produces translations out of context).